tests/integration/test_gitlab.py

```diff
@@ -246,19 +246,38 @@ def test_create_issue(self):
         """
         see class comment in case of fail
         """
+        labels = ["label1", "label2"]
         issue_title = f"New Issue {self.random_str}"
         issue_desc = f"Description for issue {self.random_str}"
-        issue = self.project.create_issue(title=issue_title, description=issue_desc)
+        issue = self.project.create_issue(
+            title=issue_title, body=issue_desc, labels=labels
+        )
+
+        assert issue.title == issue_title
+        assert issue.description == issue_desc
+        for issue_label, label in zip(issue.labels, labels):
+            assert issue_label.name == label
+
+    def test_create_private_issue(self):
+        """
+        see class comment in case of fail
+        """
+        issue_title = f"New Confidential Issue {self.random_str}"
+        issue_desc = f"Description for issue {self.random_str}"
+        issue = self.project.create_issue(
+            title=issue_title, body=issue_desc, private=True
```

Comment:
Can you please create also tests for other forges that will check that we are correctly raising an error?

Reply:
Yes, added it [here](https://github.com/packit-service/ogr/pull/441/files#diff-6f042b301fdcde9c4256f0b7bfe15798R411-R412 )for Github, Already added tests for pagure for private issue.